README.md

```diff
@@ -19,6 +19,9 @@ const multi = require('apostrophe-multisite')({
 
   shortNamePrefix: process.env.SHORTNAME_PREFIX || 'multisite-',
 
+  //Specifically with cloud hosting, servers are not very necessary, so directly setting the port is also possible. When using Heroku it can be set via PORT env var.
```

Comment:
This change should be made in the module itself so this option becomes unnecessary. The environment variable should win.

Reply:
Ah i see. In my heroku solution i am setting env variables directly through heroku website, and not with heroku CLI, so it is (as far as i experienced) necessary to call those in the code. That is why i thought this was relevant. 

Should i revert the commit, or just let you pick what you want?